dffml/operation/deploy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine
```

Comment:
Because we may have to start the container if it hasn't been started, we should require that Dockerfiles have a `Usage` statement. From when you see `docker run` until the FROM or a comment with a blank line (`#` on a line with nothing else), assume that it's the docker command, parse it using [`shlex`](https://docs.python.org/3/library/shlex.html?highlight=shlex#shlex.split). We'll use this as the docker command to run.


```suggestion
# Usage:
# docker run ...
# 
FROM alpine
```